networkx/algorithms/threshold.py

```diff
@@ -88,7 +94,14 @@ def creation_sequence(degree_sequence, with_labels=False, compact=False):
 
     with_labels and compact cannot both be True.
 
-    Returns None if the sequence is not a threshold sequence
+    Parameters
```

Comment:
This should include all of the parameters, including `with_labels` and `compact`. Note that the info that's currently in the `Notes` section could probably be used as the parameter descriptions instead.